components/promotion/base/appstudio-pipelines-runner-rolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: appstudio-pipelines-runner-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: appstudio-pipelines-runner
+subjects:
+- kind: ServiceAccount
+  name: rhtap-promotion-pipeline
```

Comment:
Why not reuse the `appstudio-pipeline` service account and give it the `appstudio-gitops-service-argocd-argocd-application-controller` role?